app/controllers/AppealsController.scala

```diff
@@ -215,11 +219,12 @@ class AppealsController @Inject()(val appConfig: AppConfig,
                 case "AWS4-HMAC-SHA256" => "SHA-256"
                 case _ => throw new Exception("[AppealsController][createSDESNotifications] failed to recognise Checksum algorithm")
               }
+              val sanitisedFileName: String = details.fileName.replaceAll(regexToRemoveSpacesAndControlCharacters,"_").replaceAll(regexToSanitiseFileName, "_")
               SDESNotification(
                 informationType = appConfig.SDESNotificationInfoType,
                 file = SDESNotificationFile(
                   recipientOrSender = appConfig.SDESNotificationFileRecipient,
-                  name = details.fileName,
+                  name = sanitisedFileName,
```

Comment:
I think we talked about putting this behind a feature switch so we can do some testing in QA without if needed?